pkgs/development/python-modules/Mako/default.nix

```diff
@@ -52,7 +54,7 @@ buildPythonPackage rec {
   ];
 
   disabledTestPaths = [
-    # lingua dependency is not packaged
```

Comment:
We are actually running some lingua-related tests though right? Otherwise adding it as a `checkInput` is pointless. And disabling a test because it raises a type error gives no indication that it's not Actually Broken.